packages/opencode/src/cli/cmd/tui/routes/session/index.tsx

```
@@ -994,7 +1028,8 @@ function UserMessage(props: {
   )
 }
 
-function AssistantMessage(props: { message: AssistantMessage; parts: Part[]; last: boolean }) {
+function AssistantMessage(props: { message: AssistantMessage; parts: Part[]; last: boolean; contextLimit: number }) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AssistantMessage'. Consider to delete it or rename it.

'AssistantMessage' is defined here:


(lint/suspicious/noRedeclare)